src/app/core/services/form.service.ts

```diff
@@ -40,10 +40,10 @@ export class FormService {
       return 'Account not found';
     }
     if (formGroup.hasError('incorrectPassword')) {
-      return 'Incorrect password';
+      return 'Fuck Putin';
```

Comment:
And I was wondering why account "KGB" started watching this repo.;)

Reply:
😄